packages/vaadin-combo-box/theme/lumo/vaadin-combo-box-dropdown-styles.js

```diff
@@ -2,83 +2,84 @@ import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styl
+  :host(:not([closing])) [part~='content'] {
+    min-height: calc(2 * var(--lumo-space-s) + var(--lumo-icon-size-s));
+  }
 
-    [part~='overlay'] {
-      position: relative;
-    }
+  [part~='overlay'] {
+    position: relative;
+  }
+
+  :host([loading]) [part~='loader'] {
+    box-sizing: border-box;
+    width: var(--lumo-icon-size-s);
+    height: var(--lumo-icon-size-s);
+    position: absolute;
+    z-index: 1;
+    left: var(--lumo-space-s);
+    right: var(--lumo-space-s);
+    top: var(--lumo-space-s);
+    margin-left: auto;
+    margin-inline-start: auto;
+    margin-inline-end: 0;
+    border: 2px solid transparent;
+    border-color: var(--lumo-primary-color-50pct) var(--lumo-primary-color-50pct) var(--lumo-primary-color)
+      var(--lumo-primary-color);
+    border-radius: calc(0.5 * var(--lumo-icon-size-s));
+    opacity: 0;
+    animation: 1s linear infinite lumo-combo-box-loader-rotate, 0.3s 0.1s lumo-combo-box-loader-fade-in both;
+    pointer-events: none;
+  }
 
-    :host([loading]) [part~='loader'] {
-      box-sizing: border-box;
-      width: var(--lumo-icon-size-s);
-      height: var(--lumo-icon-size-s);
-      position: absolute;
-      z-index: 1;
-      left: var(--lumo-space-s);
-      right: var(--lumo-space-s);
-      top: var(--lumo-space-s);
-      margin-left: auto;
-      margin-inline-start: auto;
-      margin-inline-end: 0;
-      border: 2px solid transparent;
-      border-color: var(--lumo-primary-color-50pct) var(--lumo-primary-color-50pct) var(--lumo-primary-color)
-        var(--lumo-primary-color);
-      border-radius: calc(0.5 * var(--lumo-icon-size-s));
+  @keyframes lumo-combo-box-loader-fade-in {
+    0% {
       opacity: 0;
-      animation: 1s linear infinite lumo-combo-box-loader-rotate, 0.3s 0.1s lumo-combo-box-loader-fade-in both;
-      pointer-events: none;
     }
 
-    @keyframes lumo-combo-box-loader-fade-in {
-      0% {
-        opacity: 0;
-      }
-
-      100% {
-        opacity: 1;
-      }
+    100% {
+      opacity: 1;
     }
+  }
 
-    @keyframes lumo-combo-box-loader-rotate {
-      0% {
-        transform: rotate(0deg);
-      }
+  @keyframes lumo-combo-box-loader-rotate {
+    0% {
+      transform: rotate(0deg);
+    }
 
-      100% {
-        transform: rotate(360deg);
-      }
+    100% {
+      transform: rotate(360deg);
     }
+  }
 
-    /* RTL specific styles */
+  /* RTL specific styles */
 
-    :host([loading][dir='rtl']) [part~='loader'] {
-      left: auto;
-      margin-left: 0;
-      margin-right: auto;
-      margin-inline-start: 0;
-      margin-inline-end: auto;
-    }
-  `,
-  { moduleId: 'lumo-combo-box-overlay', include: ['lumo-overlay', 'lumo-menu-overlay-core'] }
-);
+  :host([loading][dir='rtl']) [part~='loader'] {
+    left: auto;
+    margin-left: 0;
+    margin-right: auto;
+    margin-inline-start: 0;
+    margin-inline-end: auto;
+  }
+`;
+
+registerStyles('vaadin-combo-box-overlay', [overlay, menuOverlayCore, comboBoxOverlay], {
```

Comment:
I was wondering if this PR addresses #1697 too? Or is that still an issue with new includes?

Reply:
Seems that this PR alone doesn't solve the issue. Maybe if we added some extra logic to `registerStyles` to eliminate duplicate CSSResults etc